.github/workflows/ci.yml

```diff
@@ -34,7 +34,7 @@ jobs:
         uses: actions/checkout@v4
         with:
           path: up-conan-recipes
-          repository: gregmedd/up-conan-recipes
+          repository: eclipse-uprotocol/up-conan-recipes
```

Comment:
Minor: this change has already been made on main. I don't think it will cause any conflicts on merge, but if the merge fails I might ask you to rebase this PR to main.

Reply:
gotcha, I will remove this change from my PR to keep the scope small/relevant to valgrind